scripts/binaryninja/viz.py

```diff
@@ -0,0 +1,54 @@
+from binaryninja import *
+import os.path
+
+blue = HighlightStandardColor.BlueHighlightColor
+black = HighlightStandardColor.BlackHighlightColor
+white = HighlightStandardColor.WhiteHighlightColor
+clear = HighlightStandardColor.NoHighlightColor
+
+
+MBASE = 0x0007fffffdb7000
+
+
+def viz(view, fname, base=0):
+    """
+    Given a Manticore workspace, or trace file, highlight the basic blocks.
+    """
+    fname = fname.replace('/mnt/hgfs', '~')
```

Comment:
Yeah, this is just a convenience because I run Manticore in a ubuntu vm and binja in my mac. I can use `realpath` to get the global path in my vm and can use it directly in binja.